Doc/whatsnew/3.8.rst

```diff
@@ -769,6 +769,8 @@ Changes in the Python API
 * The :meth:`imap.IMAP4.logout` method no longer ignores silently arbitrary
   exceptions.
 
+* The :meth:`py_compile.compile` now supports silent mode.
```

Comment:
Assuming we fix up the compatibility problem, I think this belongs under “Improved” Modules, not the Porting section.

The sentence needs a noun; I suggest

The ``:func:`py_compile.compile` `` function . . .